Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleGeoTrackingSceneView.swift

```diff
@@ -162,6 +159,11 @@ public struct WorldScaleGeoTrackingSceneView: View {
                 .padding(8)
                 .background(.regularMaterial, ignoresSafeAreaEdges: .horizontal)
         }
+        .onChange(of: viewModel.scene.basemap?.loadStatus) { loadStatus in
+            withAnimation {
+                viewModel.scene.basemap?.baseLayers.forEach( { $0.opacity = 0 })
```

Comment:
This is used to set the basemap opacity to zero when the view first appears since the basemap base layers should only be visible while calibrating the elevation and heading. The baselayers are not available when the view initializes because the scene basemap has not loaded.

Reply:
I see 🫡

- Instead of setting the opacity on each status change, maybe only change it when status is `loaded`
- Animation seems not to be effective for setting the opacity of base layer, can be removed
- Since this method also sets the opacity like the other callsites in the calibration view, you can extract them into a method on the view model